pkg/flexvolume/rbd/util.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/Unknwon/goconfig"
+	"github.com/golang/glog"
+)
+
+const (
+	rbdBin = "rbd"
+)
+
+type rbdDevice struct {
+	rbdBinPath string
+	volId      string
+	pool       string
+	fstype     string
+	mappedDev  string
+}
+
+func newRbdDevice(volId, pool, fstype string) (*rbdDevice, error) {
+	rbdPath, err := exec.LookPath(rbdBin)
+	if err != nil {
+		return nil, fmt.Errorf("cannot find %s: %v", rbdBin, err)
+	}
+
+	return &rbdDevice{
+		rbdBinPath: rbdPath,
+		volId:      volId,
+		pool:       pool,
+		fstype:     fstype,
+	}, nil
+}
+
+func (r *rbdDevice) mapDevice() (string, error) {
+	glog.Infof("map device %s", r.volId)
```

Comment:
`Infof()` is too much here, how about V(3) or V(5)?

Reply:
Fixed all of them.